<ui version="4.0" >
 <class>InputDeviceEditor</class>
 <widget class="QDialog" name="InputDeviceEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Input Device</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_nameLabel" >
     <property name="text" >
      <string>Device name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="m_nameEdit" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTreeWidget" name="m_tree" >
     <column>
      <property name="text" >
       <string>Channel</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QToolButton" name="m_addButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="iconSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_editButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="iconSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_wizardButton" >
       <property name="toolTip" >
        <string>Start the device wizard that helps you get the correct channels for the device</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="iconSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>128</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputDeviceEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputDeviceEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
